packages/web-config-server/src/authSession/authSession.js

```diff
@@ -15,6 +15,7 @@ const auth = () => async (req, res, next) => {
     // if using basic or bearer auth, check credentials and set access policy for that user
     const authHeaderUser = await getUserFromAuthHeader(req);
     if (authHeaderUser) {
+      req.userJson = { userId: authHeaderUser.id };
       req.accessPolicy = await getAccessPolicyForUser(authenticator, authHeaderUser.id);
```

Comment:
It is really confused me here as we assign `userJson` and `accessPolicy` from session and auth header in this file, but I think it is required for handling permission for servers and frontend at the same time. 

I think it should be fine to assign `userJson` from header? With that `dashboards` router can access `userId` from `this.req.userJson`.

Reply:
Ahaha honestly this area of code always confuses me a bit. But yup, I like changing it so that we can reference by the `userJson` field 👍